python/pyspark/ml/tuning.py

```diff
@@ -613,7 +615,9 @@ def copy(self, extra=None):
         """
         if extra is None:
             extra = dict()
-        return TrainValidationSplitModel(self.bestModel.copy(extra))
+        bestModel = self.bestModel.copy(extra)
+        validationMetrics = self.validationMetrics
+        return TrainValidationSplitModel(bestModel, validationMetrics)
```

Comment:
It would be great if you could add test case for `TrainValidationSplitModel` copy method.